### PR TITLE
Fixes chest rig & mag bandoleer descriptions

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary

```SUMMARY: Bugfixes "corrects descriptions of chest rigs and mag bandoleers"```

#### Purpose of change
The magazine bandoleer description, `"A bandolier of pouches capable of carrying four magazines close at hand."`, was more descriptive of a chest rig. Examples below:

The chest rig description, `"A light vest covered in webbing, pockets and straps.  This variety is favored by the military, as it is capable of holding four magazines close at hand."`, was more descriptive of a tactical vest. Examples below:

Neither describes a bandoleer, pretty much a cartridge belt, which is already represented in game.

#### Describe the solution

To prevent future confusion, every instance of the item ID `chestrig` has been replaced with `tacvest`.

Every subsequent instance of the item ID `magbandolier` has been replaced with `chest rig`.

Both item descriptions were updated, the new 'tacvest' item being renamed as a tac vest (woah). Volumes and weights of both items were swapped, as the old bandoleer was heavier and larger than the old chest rig.

#### Describe alternatives you've considered
None.

#### Additional context
Neither is a plate carrier, which we don't have in game. Neither is an armored plate carrier (which is the MBR).

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -1726,7 +1726,7 @@
         ],
         "entries": [
           { "item": "glock_18c", "ammo-item": "9mmP", "container-item": "holster" },
-          { "item": "magbandolier", "contents-group": "army_mags_glock17" }
+          { "item": "tacvest", "container-group": "army_mags_glock17" }
```
Comment (LaVeyanFiend) [inline data/json/professions.json]: Given you changed this line but not the other at 1891 from ``contents-group`` to ``container-group``, one must be wrong.

File: data/json/professions.json [hunk 2]
```diff
@@ -1888,7 +1888,7 @@
           { "item": "usp_45", "ammo-item": "45_acp", "charges": 12, "container-item": "holster" },
           { "item": "usp45mag", "ammo-item": "45_acp", "charges": 12 },
           { "item": "usp45mag", "ammo-item": "45_acp", "charges": 12 },
-          { "item": "chestrig", "contents-group": [ "army_mags_m2010" ] }
+          { "item": "tacvest", "contents-group": [ "army_mags_m2010" ] }
```
Comment (LaVeyanFiend) [inline data/json/professions.json]: See previous statement

File: data/json/items/armor/ammo_pouch.json [hunk 3]
```diff
@@ -51,14 +51,14 @@
     "flags": [ "WATER_FRIENDLY", "BELTED" ]
   },
   {
-    "id": "magbandolier",
```
[the remark this answers is not in the document]
Comment (tenmillimaster) [reply data/json/items/armor/ammo_pouch.json]: Oh yes.

File: data/json/professions.json (hunk 1 above)
Comment (tenmillimaster) [reply data/json/professions.json]: Odd. Not sure how that happened.

Comment (LaVeyanFiend) [reply data/json/professions.json]: Um, you not gonna change it or the other one?

Comment (tenmillimaster) [reply data/json/professions.json]: Forgot to push.

**Comment by ZhilkinSerg** [comment]
DEBUG    : invalid recipe id "magbandolier"
 
 FUNCTION : const T& string_id<T>::obj() const [with T = recipe]
 FILE     : src/recipe_dictionary.cpp
 LINE     : 59

**Comment by ZhilkinSerg** [comment]
You need to obsolete `magbandolier` recipe with:

```
  {
    "type": "recipe",
    "result": "magbandolier",
    "obsolete": true
  }
```

in `/data/json/recipes/recipe_obsolete.json`.

**Pull request merged**